src/amp-story-player/amp-story-player.js

```diff
@@ -14,9 +14,13 @@
  * limitations under the License.
  */
 
+import {AmpStoryPlayer} from './amp-story-player-impl';
 import {AmpStoryPlayerManager} from './amp-story-player-manager';
 
 self.onload = () => {
   const manager = new AmpStoryPlayerManager(self);
   manager.loadPlayers();
 };
+
+// eslint-disable-next-line no-undef
+globalThis.ampStoryPlayer = AmpStoryPlayer;
```

Comment:
Nit: uppercase A as it's usually good practice to start constructors with a capital letter